Templates/Files/Local Files/Upload files from local fileshare to Azure Data Lake/long-description.md

```diff
@@ -0,0 +1,24 @@
+This Process uploads files from a **local filesystem** based on a **file mask** and a **source path** to **Azure Data Lake**.
+
+![Template](assets/Upload_files_from_local_fileshare_to_Azure_Data_Lake.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Frends agent has access to the local files that need to be uploaded.
+- The Azure user has the permission to upload files to the Data Lake.
```

Comment:
Should "the permission" be "rights" instead in this context?

Reply:
Whatever word you choose - it should be plural. So either permissions or rights.